Maestro/subscriptions.json

```diff
@@ -322,7 +322,8 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/uwp6.2/Latest.txt",
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/uwp6.2/Latest.txt"
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/uwp6.2/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/master/Latest.txt"
```

Comment:
This doesn't match https://github.com/dotnet/coreclr/blob/release/uwp6.2/dependencies.props currently, do you have changes planned there? (Same difference in CoreFX with https://github.com/dotnet/corefx/blob/release/uwp6.2/dependencies.props.)

release/uwp6.1 dependencies.props appear to be set up ok, for reference.

Reply:
I still need to update those...let me finish those two repos and I'll swing back around to this PR